apps/docs/src/app/experimental/component-docs/tabs/tabs-docs.module.ts

```diff
@@ -38,7 +39,8 @@ const routes: Routes = [
         ExperimentalFormModule,
         ExperimentalSwitchModule,
         ExperimentalButtonModule,
-        ExperimentalSearchModule
+        ExperimentalSearchModule,
+        ExperimentalTagModule
```

Comment:
Why do you need it in tabs docs?

Reply:
removed. This is valid for the rest of the components tho :)